core/src/global.ts

```diff
@@ -1,18 +1,43 @@
+import type {
+  CapacitorInstance,
+  WindowCapacitor,
+} from './definitions-internal';
 import { legacyRegisterWebPlugin } from './legacy/legacy-web-plugin-merge';
-import { initCapacitorGlobal } from './runtime';
 import type { WebPlugin } from './web-plugin';
 
-export const Capacitor = /*#__PURE__*/ initCapacitorGlobal(
-  (typeof globalThis !== 'undefined'
-    ? globalThis
+// eslint-disable-next-line
+const createCapacitor = require('../native-bridge');
```

Comment:
I think this is importing the whole native-bridge into core, which makes the native code to ignore the native-bridge.js file and still use the core code

Reply:
Yeah, I don't think its working as expected...I mostly did that so the tests would pass. I will keep tinkering with the tests so we don't have to require the bridge like here